bin/v2.js

```diff
@@ -525,8 +555,111 @@ const releaseCmd = (cmdObj, bucketNames, lambdaFunctions, environments, bucketPa
     .catch(err => console.error(err));
 }
 
+const cleanupCmd = async (cmdObj, bucketNames, lambdaFunctions, environments, bucketPath, packageName) => {
+
+  const now = new Date();
+  const ninetyDaysAgo = new Date(now.getTime() - (1000 * 60 * 60 * 24 * 90));
+
+
+  // This line might be too cleaver. It's getting all the server environments and then getting an array of just the unique ones.
+  // noinspection JSUnresolvedVariable
+  const serverEnvironments = [...new Set(environments.map(environment => environment.serverEnvironment))];
```

Comment:
👀

Reply:
I don't know what the `...` part does, but getting a collection of unique items by converting things to a set is not especially unusual